build_scripts/build_runner.py

```diff
@@ -889,23 +889,13 @@ def main():
 Use this argument if you want to specify repository
 which is not present in mediasdk_directories.''')
     parser.add_argument('-b', "--build-type", default='release',
-                        choices=['release', 'debug'],
+                        choices=Build_type,
```

Comment:
It will not work. You need values of enum. [type.value for type in Build_type].
The same in all other places like this.